### PR TITLE
Tailsitter attitude control

### PR DESCRIPTION
**Pull request by jingego**
Revised transition start pitch angle to current vehicle's attitude instead of setpoint.
Fixed Roll control in fixed-wing frame during transition.

[Flight Log Link](https://www.dropbox.com/s/3f8y0uadqdyjly8/02_59_44.ulg?dl=0)

### CODE REVIEW

File: src/modules/vtol_att_control/tailsitter.cpp [hunk 1]
```diff
@@ -272,14 +274,6 @@ void Tailsitter::update_transition_state()
 		_mc_roll_weight = 1.0f;
 		_mc_pitch_weight = 1.0f;
 
-		// check front transition timeout
```
Comment (dagar) [inline src/modules/vtol_att_control/tailsitter.cpp]: Why remove the transition timeout?

Comment (jingego) [inline src/modules/vtol_att_control/tailsitter.cpp]: There's no timeout part when I revised the code... 
Do I need revise it again base on the current code and then pull request?

this is my first time to pull request...not familiar it at all... sorry for causing any trouble.

**Comment by TSC21** [comment]
If you want to pr against upstream master you should track it. Please rebase on current master by pulling it or making changes on upstream master and pr again.

**Comment by jingego** [comment]
I will pr it again. thx!

**Pull request closed without merging**